scripts/data-statistics.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const bcd = require('..');
+
+const browsers = ['chrome', 'chrome_android', 'edge', 'firefox', 'ie', 'safari', 'safari_ios', 'webview_android'];
+let stats = { total: { all: 0, true: 0, null: 0, real: 0 } };
+browsers.forEach(browser => {
+  stats[browser] = { all: 0, true: 0, null: 0, real: 0 }
+});
+
+const processData = (data) => {
+  if (data.support) {
+    browsers.forEach(function(browser) {
+      stats[browser].all++;
+      stats.total.all++;
+      let real_value = true;
+      if (!data.support[browser]) {
+        stats[browser].null++;
+        stats.total.null++;
+        real_value = false;
+      } else {
+        if (JSON.stringify(data.support[browser]).includes('null')) {
```

Comment:
It looks like this will match on any occurrence of "null" in the JSON, which includes "null" in notes. Should this not iterate the support statements instead?

Reply:
Good point! I guess the motivation here was to catch all `null` values in both `version_added` and `version_removed` plus them appearing multiple times in support arrays.
I'll look into this. Probably need to iterate into this to be precise.

Reply:
Heh, this is great! Throughout all data this caused 34 wrong `null` values ("null" appearing in notes) and 1407 wrong `true` values ("true" appearing in notes and in flags as the data often has "`value_to_set: 'true'`").

Reply:
Updated. Can you take another look?